docs/use/Interact-with-dApps/index.md

```diff
@@ -3,18 +3,44 @@ sidebar_position: 4
+
+## Evmos dApps
+
+Alongside the applications built by the developer community, the Evmos Core Development Team also builds its own [Evmos dApps](https://app.evmos.org). An easy way to get started is by staking and voting for governance proposals.
+
+### Staking
+
+Staking plays an essential role in keeping the Evmos network secure and decentralized. The Evmos network is run by a technology that incentivizes a set of community members to keep it online and decentralized, called Proof-of-Stake. These community members, called validators, are responsible for validating transactions in a block and are usually very technical teams as they have to monitor and upgrade the software that runs the Evmos network. Validators are incentivized to stay online as they earn a commission on your stake.
+
+You can help keep the network secure and put your EVMOS tokens at "stake", by delegating them to a validator. This will earn you a daily interest in form of new EVMOS tokens (aka. staking rewards). As long as your tokens are staked, you cannot transfer or use them in other dApps. At any given time, you can undelegate your tokens. This will allow you to start using your tokens again after an unbonding period that is defined in the protocol (e.g. two weeks).
+
+You can choose validators to stake your tokens with from a list on the [Evmos Staking dApp](https://app.evmos.org/staking). We recommend choosing a reputable validator, that can promise to stay online and doesn't have a large voting power. Validators that are offline for an extended period or act maliciously can get slashed, resulting in a loss of your stake. Also, delegating to validators with less voting power keeps the network more decentralized. You can switch between validators without undelegating your tokens by redelegating them.
+
+### Governance
+
+Evmos has an on-chain governance mechanism for making changes to the network protocol, such as changing chain parameters (like the unbonding period for staking), spending funds from the community pool (e.g. to fund teams to build on Evmos), or upgrading the Evmos software that is run by validators.
+
+Developers propose changes through code updates and the community votes on whether to accept or reject the proposed change. Every participant can vote for proposals, as long as they have staked EVMOS tokens. Each vote is weighted by the amount of staked tokens.
+
+Evmos is known for hosting one of the most active governance communities in the blockchain space. As a decentralized autonomous organization (DAO) the ownership, norms, and culture of the Evmos network are thus spread amongst the community. Once you have staked your tokens, head over to the [Evmos Governance dApp](https://app.evmos.org/governance) to start voting on governance proposals.
```

Comment:
For the first sentence, I would mention Commonwealth and link to our CM site. I do not fully understand the second sentence (`As a decentralized autonomous organization (DAO) the ownership, norms, and culture of the Evmos network are thus spread amongst the community.`). I think the DAO just got formed but not finalized. It would be good to also link to our DAO (https://evmos.community/)